### PR TITLE
scared

### PR DESCRIPTION
**Pull request by elanawood**
I made some cool changes

### CODE REVIEW

File: server/mustache-templates/navPartial.mst [hunk 1]
```diff
@@ -3,7 +3,7 @@
     <button class="navbar-toggler navbar-dark" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
         <span class="navbar-toggler-icon"></span>
     </button>
-
+    <!-- Delilah needs more treats -->
```
Comment (micahdbak) [inline server/mustache-templates/navPartial.mst:6]: Very cool and very important feature. Thank you for making these changes. Approved!

**Review by micahdbak — COMMENTED** [review]
Please deploy to the dev server and mention that you did so in your test plan before merging into main

File: server/mustache-templates/navPartial.mst (hunk 1 above)
Comment (johnny-brave) [reply server/mustache-templates/navPartial.mst:6]: I agree

**Comment by micahdbak** [comment]
Please merge the new changes from main into this branch; you can do this with the following commands:

```
git checkout main
git pull origin main
git checkout ew-testingsofun
git merge main
git pull origin ew-testingsofun
```

**Comment by micahdbak** [comment]
(This PR was for testing purposes; I'm going to close it now)

**Pull request closed without merging**